app/controllers/welcome_controller.rb

```diff
@@ -28,6 +29,22 @@ def mentor
     @report = params[:report].blank? ? "basics" : ERB::Util.html_escape(params[:report])
   end
 
+  def participant
+    if @current_user.person_type == "Participant"
+      @participant = @current_user.person
+      if @participant && !@participant.completed_intake_form?
+        redirect_to participant_signup_intake_form_path
+      end
+      @person = current_user.person
+      @events = @person.event_attendances.future_attending.collect(&:event)
+      @event_groups = EventGroup.where(allow_external_volunteers: true)
```

Comment:
This doesn't seem like the correct set of `EventGroups` to be displaying, I think it should be `allow_external_students: true`